jekyll_files/docs/admin-documentation/register-app-in-laserfiche.md

```diff
@@ -21,7 +21,7 @@ parent: Laserfiche SharePoint Integration Administration Guide
    <a href="../assets/images/clientIdRegistered.png"><img src="../assets/images/clientIdRegistered.png"></a>
 1. One way or another, an app with that client ID should now exist. Open the app and switch from the general tab to the authentication tab.
    <a href="../assets/images/redirectUri.png"><img src="../assets/images/redirectUri.png"></a>
-1. Copy the URLs from your three SharePoint Pages with the Laserfiche web part and add them in the developer console as new redirect URIs. Each URI should end in `.aspx`
+1. Copy the URLs from your SharePoint page with the Laserfiche sign in web part, and append the suffix `?autologin` and add it in the developer console as a new redirect URI. The URI should end in `.aspx?autologin`
```

Comment:
Copy the URL from your SharePoint LaserficheSignIn page, append the query parameter `?autologin`  and add it to the developer console as a new redirect URI. The URI should end with `SitePages/LaserficheSignIn.aspx?autologin`.

Also update the screenshot